release/release_gold.sh

```diff
@@ -203,6 +215,152 @@ function print_help {
 	exit "${LIFERAY_COMMON_EXIT_CODE_HELP}"
 }
 
+function reference_new_releases {
+	if [[ "${_PRODUCT_VERSION}" != *q* ]]
+	then
+		lc_log INFO "Skipping the update to the references in the liferay-jenkins-ee repository."
+
+		return "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}"
+	fi
+
+	prepare_branch_to_commit_from_master "${_PROJECTS_DIR}/liferay-jenkins-ee/commands" "new_releases_branch"
+
+	if [ "${?}" -ne 0 ]
+	then
+		lc_log ERROR "Unable to prepare the next release references branch."
+
+		return "${LIFERAY_COMMON_EXIT_CODE_BAD}"
+	fi
+
+	local base_url="http://mirrors.lax.liferay.com/releases.liferay.com"
+
+	local previous_product_version="$(grep "portal.latest.bundle.version\[master\]=" "build.properties" | cut -d "=" -f 2)"
+
+	for component in osgi sql tools
+	do
+		add_property \
+			"build.properties" \
+			"portal.${component}.zip.url\[${_PRODUCT_VERSION}\]" \
+			"${base_url}/${LIFERAY_RELEASE_PRODUCT_NAME}/${_PRODUCT_VERSION}/liferay-${LIFERAY_RELEASE_PRODUCT_NAME}-${component}-${_PRODUCT_VERSION}-${LIFERAY_RELEASE_RC_BUILD_TIMESTAMP}.zip" \
+			"portal.${component}.zip.url\[${previous_product_version}\]="
+	done
+
+	add_property \
+		"build.properties" \
+		"plugins.war.zip.url\[${_PRODUCT_VERSION}\]" \
+		"http://release-1/1/userContent/liferay-release-tool/7413/plugins.war.latest.zip" \
+		"plugins.war.zip.url\[${previous_product_version}\]="
+
+	add_property \
+		"build.properties" \
+		"	portal.bundle.tomcat\[${_PRODUCT_VERSION}\]" \
+		"${base_url}/${LIFERAY_RELEASE_PRODUCT_NAME}/${_PRODUCT_VERSION}/liferay-${LIFERAY_RELEASE_PRODUCT_NAME}-tomcat-${_PRODUCT_VERSION}-${LIFERAY_RELEASE_RC_BUILD_TIMESTAMP}.7z" \
+		"portal.bundle.tomcat\[${previous_product_version}\]="
+
+	add_property \
+		"build.properties" \
+		"portal.license.url\[${_PRODUCT_VERSION}\]" \
+		"http://www.liferay.com/licenses/license-portaldevelopment-developer-cluster-7.0de-liferaycom.xml" \
+		"portal.license.url\[${previous_product_version}\]="
+
+	add_property \
+		"build.properties" \
+		"portal.version.latest\[${_PRODUCT_VERSION}\]" \
+		"${_PRODUCT_VERSION}" \
+		"portal.version.latest\[${previous_product_version}\]="
+
+	add_property \
+		"build.properties" \
+		"portal.war.url\[${_PRODUCT_VERSION}\]" \
+		"${base_url}/${LIFERAY_RELEASE_PRODUCT_NAME}/${_PRODUCT_VERSION}/liferay-${LIFERAY_RELEASE_PRODUCT_NAME}-${_PRODUCT_VERSION}-${LIFERAY_RELEASE_RC_BUILD_TIMESTAMP}.war" \
+		"portal.war.url\[${previous_product_version}\]="
+
+	add_property \
+		"build.properties" \
+		"portal.latest.bundle.version\[${_PRODUCT_VERSION}\]" \
+		"${_PRODUCT_VERSION}" \
+		"portal.latest.bundle.version\[${previous_product_version}\]="
+
+	replace_property \
+		"build.properties" \
+		"portal.latest.bundle.version\[master\]" \
+		"${_PRODUCT_VERSION}" \
+		"portal.latest.bundle.version\[master\]=${previous_product_version}"
+
+	local previous_quarterly_release_branch_name="$(grep "portal.latest.bundle.version" "build.properties" | tail -1 | cut -d '[' -f 2 | cut -d ']' -f 1)"
```

Comment:
@kiwm, can you please format it like you did above for the functions `add_property` and `replace_property`?